### ISSUE TITLE
Rendering a object instead string

### ISSUE THREAD

**Issue by darlanmendonca** [lingui/js-lingui#819]
**Describe the bug**
Rendering a `[object Object]` instead of string

**To Reproduce**
Occurs when I use <Trans> inside a placeholder prop, e.g. 

```
<input placeholder={<Trans>Username</Trans>} />
```

**Expected behavior**
Translated string or default string (key)

**Comment by semoal** [comment]
In this cases is recommended to use t`Username` macro, because placeholder internally is not rendering as a normal jsx component.

Anyways, probably we should log something more descriptive or making it work, i don't know

**Comment by darlanmendonca** [comment]
@semoal I'm already using macro

```
import { Trans } from '@lingui/macro'

<input placeholder={<Trans>Username</Trans>} />
```

**Comment by tricoder42** [comment]
Not sure if we can do something about it. The same thing would happen if you use any other JSX component:

```
// won't work either
<input placeholder={<p>Username</p>} />
```

TLDR: Don't use JSX in props which expects string. Use `t` macro instead:

```
<input placeholder={t`Username`} />
```

**Comment by darlanmendonca** [comment]
@tricoder42 `t`macro produces the same result :(

```
<input placeholder={t`Username`} />
```

```
<input placeholder="[object Object]" />
```

**Comment by semoal** [comment]
Could you provide a minimal setup? Like a codesandbox or stackblitz? probably is something in your babel conf that is not correctly set

**Comment by tricoder42** [comment]
I've checked it in Codesandbox and it works: https://codesandbox.io/s/nextjs-lingui-djzqm

**Comment by darlanmendonca** [comment]
Hey folks!!
I updated my version from 2.9.1 to 3.0.3 and works!
Thanks @tricoder42 by your codesandbox, I use it as reference together to documentation.
